src/screens/Accessibility/index.css

```diff
@@ -3,29 +3,36 @@ input[type='range'] {
   max-width: 513px;
 }
 
-.Accessibility > .settingsWrapper > .setting > .is-drop-down {
-  background-position: calc(100% - 20px) calc(1em + -4px),
-    calc(100% - 15px) calc(1em + -4px), calc(101% - 2.5em) calc(0.5em + -8px);
```

Comment:
changed the main background properties so it works in smaller and bigger selects so this is not needed